wasm-hello/README.md

```diff
@@ -0,0 +1,280 @@
+# WASM Hello World on Unikraft
+
+Build and run a WebAssembly (Wasm) hello world program on Unikraft.
+This program runs inside an iwasm Intel runtime environment, which is a lightweight WebAssembly runtime designed for efficient execution of WebAssembly modules.
+Follow the instructions below to set up, configure, build and run the program.
+Make sure you installed the [requirements](../README.md#requirements).
+
+## Quick Setup (aka TLDR)
+
+For a quick setup, run the commands below.
+Note that you still need to install the [requirements](../README.md#requirements).
+Before everything, make sure you run the [top-level `setup.sh` script](../setup.sh).
+
+To build and run the application for `x86_64`, use the commands below:
+
+```console
+./setup.sh
+make distclean
+wget -O /tmp/defconfig https://raw.githubusercontent.com/unikraft/catalog-core/refs/heads/scripts/wasm-hello/scripts/defconfig/qemu.x86_64
+UK_DEFCONFIG=/tmp/defconfig make defconfig
+make -j $(nproc)
+rm -f initrd.cpio
+./workdir/unikraft/support/scripts/mkcpio initrd.cpio ./rootfs/
+qemu-system-x86_64 \
+    -nographic \
+    -m 8 \
+    -cpu max \
+    -kernel workdir/build/wasm-hello-x86_64 \
+    -append "wasm-hello-x86_64 vfs.fstab=[ \"initrd0:/:extract::ramfs=1:\" ] -- main.wasm" \
+    -initrd ./initrd.cpio
+```
+
+This will configure, build and run the application, resulting in a
+`Hello world!
+buf ptr: 0x400002b0
+buf: 1234`
```

Comment:
```suggestion

\```text
Hello world!
buf ptr: 0x400002b0
buf: 1234
\```

```

(without the backslashes, I added them just so github doesn't get confused)